tensorboard/defs/defs.bzl

```diff
@@ -45,10 +45,19 @@ def tf_ts_config(**kwargs):
 
     ts_config(**kwargs)
 
-def tf_ts_library(**kwargs):
-    """TensorBoard wrapper for the rule for a TypeScript library."""
+def tf_ts_library(strict_checks = True, tsconfig = None, **kwargs):
+    """TensorBoard wrapper for the rule for a TypeScript library.
+
+    Args:
+      strict_checks: whether to enable stricter type checking. Default is True.
+          Please use `strict_checks = False` for only Polymer based targets.
+      tsconfig: TypeScript configuration. If specified, it overrides strict_checks.
+      **kwargs: keyword arguments to ts_library build rule.
+    """
+    if not strict_checks and not tsconfig:
```

Comment:
Optional, this might be slightly clearer as:

```python
if not tsconfig:
  tsconfig = "//:tsconfig.json" if strict_checks else "//:tsconfig-lax"
ts_library(tsconfig = tsconfig, **kwargs)
```